src/compiler/scala/tools/nsc/symtab/classfile/Pickler.scala

```diff
@@ -91,6 +101,31 @@ abstract class Pickler extends SubComponent {
       }
     }
 
+    override def run(): Unit = {
+      try super.run()
+      finally closeSigWriter()
+    }
+
+    private def writeSigFile(sym: Symbol, pickle: PickleBuffer): Unit = {
+      sigWriter match {
+        case Some((writer, sigOutput)) =>
+          val binaryName = sym.javaBinaryNameString
+          val binaryClassName = if (sym.isModule) binaryName.stripSuffix(nme.MODULE_SUFFIX_STRING) else binaryName
+          val relativePath = java.nio.file.Paths.get(binaryClassName + ".sig")
+          val data = pickle.bytes.take(pickle.writeIndex)
+          writer.writeFile(relativePath, data, CompilationUnitPaths(sym.sourceFile, sigOutput))
+        case _ =>
+      }
+    }
+    private def closeSigWriter() = {
+      sigWriter match {
+        case Some((writer, dest)) =>
+          writer.close()
+          reporter.info(NoPosition, "[sig files written]", force = false)
```

Comment:
This output will show up under `-verbose`. Once you see that, downstream compilation could be started.

Reply:
Should we clean up `-verbose`? The output is still full of

```
[loaded class file /Users/luc/scala/scala-2.13.0/lib/scala-library.jar(scala/Function2$mcJJI$sp.class) in 0ms]
```

which seems unnecessary.

Reply:
I don't think that stops a tool turning it on to experiment with this PR, so I'd be inclined to leave it for now.